### PR TITLE
Fixed FB share double render error.

### PR DESCRIPTION
**Pull request by joelmats**
simple fix for this issue.

### CODE REVIEW

**Comment by hhsnopek** [comment]
note:
fixes #285

**Comment by hhsnopek** [comment]
hey @joelmats it seems that this change breaks the tests, could you look into why?

**Comment by joelmats** [comment]
Sure will do. Installing xcode atm which is taking a bit.

**Comment by hhsnopek** [comment]
I'd like to get this in the next release tomorrow morning :smile: - lemme know if you need any help adding the second test!

**Comment by joelmats** [comment]
@hhsnopek Ok just got back to it. I added a method in the widget to simulate an sdk load.

**Comment by joelmats** [comment]
@hhsnopek Let me know if you want me to make any changes. I am also eager to get this in :)

File: tests/steps/facebook.coffee [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = ->
   @Given /^I create a Facebook Share Button$/, () ->
-    @driver.visit("http://localhost:8000/tests/fixtures/facebook.html")
+    @driver.visit("file:///#{process.cwd()}/tests/fixtures/facebook.html")
```
Comment (hhsnopek) [inline tests/steps/facebook.coffee]: This is actually suppose to be localhost:8000 because of how the sdk gets loaded - it was an issue we ran into

Comment (hhsnopek) [inline tests/steps/facebook.coffee]: you just need to run `python -m SimpleHTTPServer` before you run the tests (this is how it's handled on travis-ci as well)

**Comment by hhsnopek** [comment]
Just making sure i'm on the right track with this, now (only if you're using the sdk), facebook sharing will load on the current page. If you're not using the sdk it will create a pop out?

**Comment by joelmats** [comment]
Correct :)

On Wed, Feb 17, 2016 at 12:31 PM, Henry Snopek <notifications@github.com>
wrote:

> Just making sure i'm on the right track with this, now (only if you're
> using the sdk), facebook sharing will load on the current page. If you're
> not using the sdk it will create a pop out?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/carrot/share-button/pull/287#issuecomment-185390531>.
>



-- 
Joel Rikiya Matsumoto

Mobile #: 650-515-7115

**Comment by hhsnopek** [comment]
Would you you be able to get the functionality of it to pop out even when the sdk is in use to match the other networks?

**Comment by joelmats** [comment]
Hmm not sure what you mean. If the SDK is loaded, facebook loads their own
version of the popup. If a user does not want to use this version, they can
just set the config to not use the sdk i think and it will fallback to the
php popup.

On Wed, Feb 17, 2016 at 12:34 PM, Henry Snopek <notifications@github.com>
wrote:

> Would you you be able to get the functionality of it to pop out even when
> the sdk is in use to match the other networks?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/carrot/share-button/pull/287#issuecomment-185391506>.
>



-- 
Joel Rikiya Matsumoto

Mobile #: 650-515-7115

**Comment by hhsnopek** [comment]
oh alright that's perfect then - if you could just separate out the sdk test from the original facebook test that would be great (then we'll have two different configs), still want to test against both :smile:

**Comment by joelmats** [comment]
Got it :)

On Wed, Feb 17, 2016 at 12:44 PM, Henry Snopek <notifications@github.com>
wrote:

> oh alright that's perfect then - if you could just separate out the sdk
> test from the original facebook test that would be great (then we'll have
> two different configs), still want to test against both [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/carrot/share-button/pull/287#issuecomment-185395543>.
>



-- 
Joel Rikiya Matsumoto

Mobile #: 650-515-7115

**Comment by joelmats** [comment]
@hhsnopek I added 2 tests since there are actually 3 cases. 1. User specifies enabled SDK, SDK loads properly, no url needed. 2. User specifies enabled SDK, SDK does not load (for some reason), url falls back to php popup. 3. User disables SDK load, SDK does not load, and url is set to the php popup. Let me know if this makes sense :)

**Comment by hhsnopek** [comment]
Beautiful - once travis is complete if you could squash the commits into one nice commit, we'll go ahead and merge this bad boy in :smiley:

**Comment by joelmats** [comment]
Rebased and squashed to 1 commit. Cheers!

**Comment by hhsnopek** [comment]
thanks again! :smile:

**Comment by joelmats** [comment]
Np. Thanks for the help and awesome plugin!

On Wed, Feb 17, 2016 at 1:15 PM, Henry Snopek <notifications@github.com>
wrote:

> thanks again! [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/carrot/share-button/pull/287#issuecomment-185407737>.
>



-- 
Joel Rikiya Matsumoto

Mobile #: 650-515-7115

**Comment by rcamposp** [comment]
Please make a new release with these changes.

Took me a couple of hours to fix it.

**Pull request merged**